backend/windmill-worker/src/pg_executor.rs

```diff
@@ -146,8 +192,55 @@ pub async fn do_postgresql(
         .into_iter()
         .map(postgres_row_to_json_value)
         .collect::<Result<Vec<_>, _>>()?);
+    RUNNING.store(false, std::sync::atomic::Ordering::Relaxed);
```

Comment:
If one of the statement with an ending `?` fails above, I guess the function will return early and the RUNNING won't be set back to false. Do we risk something having RUNNING always true?